src/components/BuildConfigsList/BuildConfigsList.tsx

```diff
@@ -0,0 +1,254 @@
+  columns = defaultColumns,
+  componentId,
+}: IBuildConifgsListProps) => {
+  const { getSortParams, getSortGroupParams } = useSorting(sortAttributes, componentId, defaultSorting);
+
+  const [isTimesSortDropdownOpen, setIsTimesSortDropdownOpen] = useState<boolean>(false);
+
+  return (
+    <>
+      <Toolbar>
+        <ToolbarItem>
+          <Filtering
+            filterOptions={getFilterAttributes(buildConfigEntityAttributes, defaultColumns)}
+            componentId={componentId}
+            defaultFiltering={defaultFiltering}
+          />
+        </ToolbarItem>
+      </Toolbar>
+
+      <ContentBox borderTop>
+        <ServiceContainerLoading {...serviceContainerBuildConfigs} title={PageTitles.buildConfigs}>
+          <TableComposable isStriped variant="compact">
+            <Thead>
+              {/**
+               * If column order is changed, the property tableColumnIndex (see sortAttributes) has to be updated.
+               * Better solution can be implemented in the future.
+               */}
+              <Tr>
+                {columns.includes(buildConfigEntityAttributes.name.id) && (
+                  <Th width={15} sort={getSortParams(sortAttributes.name.id)}>
+                    {buildConfigEntityAttributes.name.title}
+                  </Th>
+                )}
+                {columns.includes(buildConfigEntityAttributes.description.id) && (
+                  <Th width={15}>{buildConfigEntityAttributes.description.title}</Th>
+                )}
+                {columns.includes(buildConfigEntityAttributes.buildType.id) && (
+                  <Th width={10} sort={getSortParams(sortAttributes.buildType.id)}>
+                    {buildConfigEntityAttributes.buildType.title}
+                  </Th>
+                )}
+                {columns.includes(buildConfigEntityAttributes['project.name'].id) && (
+                  <Th width={15} sort={getSortParams(sortAttributes['project.name'].id)}>
+                    {buildConfigEntityAttributes['project.name'].title}
+                  </Th>
+                )}
+                {(columns.includes(buildConfigEntityAttributes.creationTime.id) ||
+                  columns.includes(buildConfigEntityAttributes.modificationTime.id)) && (
+                  <Th width={20} className="overflow-visible">
+                    <SortGroup
+                      title="Times"
+                      sort={getSortGroupParams(
+                        columns.includes(buildConfigEntityAttributes.creationTime.id)
+                          ? sortAttributes.creationTime.id!
+                          : sortAttributes.modificationTime.id!
+                      )}
+                      isDropdownOpen={isTimesSortDropdownOpen}
+                      onDropdownToggle={() => setIsTimesSortDropdownOpen(!isTimesSortDropdownOpen)}
+                    />
+                  </Th>
+                )}
+                {columns.includes(buildConfigEntityAttributes.buildStatus.id) && (
```

Comment:
After the discussion we could see that Latest Build column introduces a lot of other issues, so we decided to remove it for now. It will be implemented in new separated jira. See jira comments.